pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -151,9 +151,11 @@ func networksToHotplugWhoseInterfacesAreNotInTheDomain(vmi *v1.VirtualMachineIns
 			_, exists := indexedDomainIfaces[ifaceStatus.Name]
 			vmiSpecIface := netvmispec.LookupInterfaceByName(vmi.Spec.Domain.Devices.Interfaces, ifaceStatus.Name)
 
-			return netvmispec.ContainsInfoSource(
-				ifaceStatus.InfoSource, netvmispec.InfoSourceMultusStatus,
-			) && !exists && vmiSpecIface.State != v1.InterfaceStateAbsent && vmiSpecIface.SRIOV == nil
+			return netvmispec.ContainsInfoSource(ifaceStatus.InfoSource, netvmispec.InfoSourceMultusStatus) &&
+				ifaceStatus.PodInterfaceName != "" &&
```

Comment:
Why is it necessary? 
What happen if it doesnt take in accoung podIfaceName?
Aren't infoSource=multus check good enough?

Reply:
On a cluster that is in a steady state, the Multus infosource and podInterfaceName should be updated together.
The added condition makes sure that they are both present.

An edge case I could think of is a hotplug scenario that was performed in close proximity to a cluster upgrade - where the Multus infosource is present, but the podInterfaceName isn't.

Reply:
Sorry I dont follow why podIfaceName is a must for this check, could you elaborate on that?

Reply:
Lets assume one upgrades from v1.4 to v1.5.
v1.4 does not have `PodInterfaceName` populated for any interface (unless you enable the dynamic naming FG and then it will be populated for the primary interface).
So new components should assume `PodInterfaceName` could be unknown.

Reply:
Old VMs (prior to 1.5) will not have this condition in their virt-launcher's source code as they would run on a previous version.